### DISCUSSION TITLE
Version 1.31.2

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#920] [Announcements]
- Bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.31.2'>Version 1.31.2</a>.</em>

**Comment by sahilium** [comment]
![20240704_211514.jpg](https://github.com/MM2-0/Kvaesitso/assets/95169974/3ce1c2e0-3e02-4af3-9e37-30ca79909ec4)

Question: I am not able to hide any apps anymore, or have I got something wrong? When I choose Show in "Never", I see a hidden badge on the app, but the app icon is still visible, how to hide the app?

This has been happening since v1.31.1.

Edit: Once I launch the app, the icon is properly hidden, is this a bug?

**Reply by MM2-0** [reply]
Probably, someone else has reported a similar issue. Can you have a look at #923 and provide more details there?

**Comment by LincCodes** [comment]
Plugins not showing despite being installed on Android 13
